src/EnergyPlus/Psychrometrics.cc

```diff
@@ -1005,7 +1005,7 @@ namespace Psychrometrics {
                 if (state.dataPsychrometrics->iPsyErrIndex(iPsyTsatFnHPb) == 0) {
                     ShowWarningMessage(state, "Enthalpy out of range (PsyTsatFnHPb)");
                     if (!CalledFrom.empty()) {
-                        ShowContinueErrorTimeStamp(state, " Routine=" + CalledFrom + ',');
+                        ShowContinueErrorTimeStamp(state, format(" Routine={},", CalledFrom));
```

Comment:
Added in commas here in a couple spots that were showing ERR diffs.  They should be all clean now.